torch/_meta_registrations.py

```diff
@@ -2450,6 +2450,19 @@ def pick_memory_format():
         elif input_tensor.is_contiguous(memory_format=torch.preserve_format):
             return torch.preserve_format
 
+    # Expand 1d -> 2d.
```

Comment:
Are you sure we don't need to update https://github.com/pytorch/pytorch/blob/b7db86600a2614adc71c92ca42d359a7ac534d78/torch/_subclasses/fake_impls.py#L1024-L1026 ?